docs/changelog/2259.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Insert ``TOX_WORK_DIR``, ``TOX_ENV_NAME``, ``TOX_ENV_DIR`` and ``VIRTUAL_ENV`` environment variables for each tox
+environments environment variables to keep contract with tox version 3 - by :user:`gaborbernat`.
```

Comment:
```suggestion
Insert ``TOX_WORK_DIR``, ``TOX_ENV_NAME``, ``TOX_ENV_DIR`` and ``VIRTUAL_ENV`` into the environment variables for all tox environments to keep contract with tox version 3 - by :user:`gaborbernat`.
```

Reply:
What do you think of the updated sentence? The previous one was a bit hard to understand.

Reply:
Sure :+1:

Reply:
You are too fast 😎

Reply:
Done in https://github.com/tox-dev/tox/commit/669faa563e4ecf975ce74b9911652e5a1fa37f0c